### PR TITLE
Portfolio revamp

### PR DESCRIPTION
**Pull request by Martindn-Ha**
Revamped entire portfolio. Will add more finalized codes and improvements continuously.

### CODE REVIEW

File: portfolio/src/main/java/com/google/sps/servlets/FormHandlerServlet.java [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/form-handler")
+public class FormHandlerServlet extends HttpServlet {
+
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+
+    // Get the value entered in the form.
```
Comment (geegeex) [inline portfolio/src/main/java/com/google/sps/servlets/FormHandlerServlet.java:20]: Great work adding comments, makes things easier to read!

File: portfolio/src/main/java/com/google/sps/servlets/FormLoadServlet.java [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+while (results.hasNext()) {
+  Entity entity = results.next();
+
+  long id = entity.getKey().getId();
+  String comment = entity.getString("input");
+
+}
+
+    Gson gson = new Gson();
```
Comment (geegeex) [inline portfolio/src/main/java/com/google/sps/servlets/FormLoadServlet.java]: Keep indentations consistent.

File: portfolio/src/main/webapp/contact.html [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+        <input type="submit" />
+      </form>
+      
+      <!--<button onclick="helloWorld();">
+        Say Hello
+      </button>
+      <p id="hello-container"></p>
+      <p>Click here to get a random message:</p>
+      <button onclick="addRandomGreeting()">Hello</button>
+      <div id="greeting-container"></div> -->
```
Comment (geegeex) [inline portfolio/src/main/webapp/contact.html]: It is best practice to remove code that is not used.

**Review by geegeex — CHANGES_REQUESTED** [review]

File: portfolio/src/main/java/com/google/sps/servlets/FormLoadServlet.java (hunk 2 above)
Comment (Martindn-Ha) [reply portfolio/src/main/java/com/google/sps/servlets/FormLoadServlet.java]: resolved.

File: portfolio/src/main/webapp/contact.html (hunk 3 above)
Comment (Martindn-Ha) [reply portfolio/src/main/webapp/contact.html]: resolved.

**Review by geegeex — APPROVED** [review]

**Pull request merged**